ttl/hbp_macaca_fascicularis_brain_ontology.ttl

```diff
@@ -0,0 +1,657 @@
+### WARNING ###
+### THIS ONTOLOGY IS TEMPORARY DO NOT USE IDENTIFIERS OUTSIDE DEV
+### WARNING ###
+@prefix : <http://www.hbp.FIXME.org/hbp_human_brain_ontology/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix nsu: <http://www.FIXME.org/nsupper#> .
+@prefix MFBA: <https://scalablebrainatlas.incf.org/services/listregions.php?template=MERetal14> .
```

Comment:
Add a '/' character at the end of the MFBA namespace: https://scalablebrainatlas.incf.org/services/listregions.php?template=MERetal14